vocabularies/earth-science-data-category.ttl

```diff
@@ -170,43 +238,67 @@ esdc:mineralogy-and-crystallography a skos:Concept ;
     skos:notation "040306" ;
     skos:prefLabel "Mineralogy and Crystallography"@en .
 
-esdc:palaeontology a skos:Concept ;
+esdc:gemmology a skos:Concept ; 
+    skos:altLabel "gemology"@en , 
+        "gems"@en ,
+        "gemstones"@en ;
```

Comment:
```suggestion
    skos:altLabel "gemology"@en 
    skos:hiddenLabel "gems"@en ,
        "gemstones"@en ;
```

Gems and Gemstones are keywords rather than names for Gemmology. I'm not a fan of having them even as Hidden Labels, but I see how they directly relate to the -ology that describes them so i'm ok to move them as such.